apps/language_server/lib/language_server/providers/code_lens/test.ex

```diff
@@ -89,11 +91,21 @@ defmodule ElixirLS.LanguageServer.Providers.CodeLens.Test do
           describe.body_scope_id == scope_id
         end)
 
-      %{"name" => test_name} =
-        ~r/^\s*test "(?<name>.*)"(,.*)?/
-        |> Regex.named_captures(Enum.at(source_lines, line - 1))
-
-      %TestBlock{name: test_name, describe: describe, line: line}
+      test_name =
+        source_lines
+        |> Enum.at(line - 1)
+        |> String.trim()
+        |> case do
+          "test " <> rest ->
+            String.split(rest, ~r/"/)
+            |> Enum.at(1)
+
+          "doctest" <> _ = line ->
+            String.split(line, ~r/,/)
+            |> Enum.at(0)
```

Comment:
Commit to the pipeline:
```suggestion
            line
            |> String.split(~r/,/)
            |> Enum.at(0)
```

Reply:
Any reason you're using a regex there rather than just splitting on "," ?